compose/animation/animation-core/src/jbMain/kotlin/androidx/compose/animation/core/AtomicActual.kt

```diff
@@ -18,4 +16,14 @@
 
 package androidx.compose.animation.core
 
-internal actual typealias AtomicReference<V> = java.util.concurrent.atomic.AtomicReference<V>
\ No newline at end of file
+import kotlinx.atomicfu.atomic
+
+internal actual class AtomicReference<V> actual constructor(value: V) {
```

Comment:
I see the source set here named `jbMain`.  I understand that we added `jbMain` in runtime module, because `skikoMain` name doesn't fit runtime purpose and it doesn't depend on skiko.

Since animation-core depends on ui, then it depends on skiko:
`implementation(project(":compose:ui:ui"))`

And the name can be skikoMain to not introduce a new name beside runtime.  Not a big thing though. No skiko code is used here directly, so `skikoMain` name here is arguable.